src/serializer.py

```diff
@@ -26,40 +26,40 @@ def get_base_dict(self):
         """
         fetches the base template
         """
-        return self.load_json("templates/base.json")
+        return self.load_json("../templates/base.json")
```

Comment:
Using the prev directory specifier `../` depends on the current working directory. 

If the user is executing from `/` this will not work.

If the user is executing from 'src/` the will work. 

Using an argparser and generating absolute paths during runtime should fix this.

Reply:
Updated it. --path and --csv have been added